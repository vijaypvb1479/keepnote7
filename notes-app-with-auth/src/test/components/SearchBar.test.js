import React from 'react';
import {shallow } from 'enzyme';
import SearchBar from '../../components/SearchBar';

import {createSerializer} from 'enzyme-to-json';
 
expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

test('test Search Bar', () => {

    const wrapper = shallow(<SearchBar />);
    expect(wrapper).toMatchSnapshot();


});