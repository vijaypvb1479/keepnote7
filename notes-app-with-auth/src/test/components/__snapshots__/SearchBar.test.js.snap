// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Search Bar 1`] = `
<SearchBar
  classes={
    Object {
      "inputInput": "SearchBar-inputInput-4",
      "inputRoot": "SearchBar-inputRoot-3",
      "search": "SearchBar-search-1",
      "searchIcon": "SearchBar-searchIcon-2",
    }
  }
/>
`;
